permissions/plugin-ws-ctm.yml

```diff
@@ -0,0 +1,9 @@
+---
+name: "ws-execution-manager"
```

Comment:
This should match the plugin ID. Please see this repo's readme.

Reply:
@daniel-beck  Thank you Daniel for helping with this.   Wow - I don't know how that ended up with that ID.  I wasn't being careful enough.  Thank you again.